subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync.c

```diff
@@ -102,6 +112,9 @@ static int init_reset(void)
 static int prepare_cb(struct lll_prepare_param *p)
 {
 	struct node_rx_pdu *node_rx;
+#if defined(CONFIG_BT_CTLR_DF_SCAN_CTE_RX)
+	struct lll_df_sync_cfg *cfg;
+#endif /* CONFIG_BT_CTLR_DF_SCAN_CTE_RX */
```

Comment:
```suggestion
```

Reply:
Moved `cfg` definition down, just before it is used in function flow.